src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -110,3 +110,12 @@ def _make_tags(replay_id, url, replay_event):
         tags.update(replay_event["tags"])
 
     return tags
+
+
+def _make_clicked_element(node):
+    element = node.get("tagName", "")
+    if "attributes" in node:
+        for key, value in node["attributes"].items():
```

Comment:
Nit:
```suggestion
        str_builder = [element]
        for key, value in node["attributes"].items():
            str_builder.append(f'[{key}="{value}"]')
        return ''.join(str_builder)
```
Pretty nit but this is the preferred/most efficient way to build a string in a loop. Avoids O(n^2) str concatenation 🤓